modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,296 +1,140 @@
-
-Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements that are available in this release.
-
-[#release-600]
-== Release 6.0
-
-Couchbase Server 6.0 was released in October 2018.
-
-[#changes-in-behavior-600]
-=== Major Behavior Changes
-
-There are a few significant changes to the production-ready version of the Analytics service.
-You'll find this information handy if you've used the Analytics service in any of the Developer Preview versions of Couchbase Server.
-
-* Bucket association is now automatic
-+
-The Developer Preview versions required you to explicitly associate the Couchbase Server buckets and Analytics buckets.
-For example:
-+
-----
-CREATE BUCKET ha WITH {"name":"health"}
-----
-+
-The Analytics service buckets now automatically mirror Couchbase Server buckets and no special configuration is required to be able to create a dataset on a bucket.
-
-* Dataset definitions no longer require the SHADOW keyword
-+
-The early Developer Preview releases (earlier than Couchbase Server 5.5) required the SHADOW keyword.
-+
-The Analytics service (Developer Preview version) was integrated with Couchbase Server during release 5.5 and the SHADOW keyword was made optional.
-+
-Starting with this release, using the SHADOW keyword is no longer allowed, and using it will return an error.
-+
-----
-CREATE DATASET ((dataverse_name.)? dataset_name)? ON bucket_name (WHERE where_exp)?.
-----
-
-* `CONNECT/DISCONNECT BUCKET` have been replaced by `CONNECT/DISCONNECT LINK`
-+
-The `CONNECT` and `DISCONNECT` commands would activate or deactivate the dataset processing of data from Couchbase Server buckets.
-+
-You can now use the following syntax to connect all datasets in a dataverse:
-+
-----
-CONNECT LINK (dataverse_name.)? Local (, (dataverse_name.)? Local)*
-----
-
-[#deprecation-600]
-=== Deprecated Platforms
-
-Support for the following platforms will be removed in a future release:
+= Release Notes for Couchbase Server 6.5 Beta
 
-* CentOS 6
-* macOS 10.11 (El Capitan)
-* Red Hat Enterprise Linux 6
-* SUSE Linux Enterprise Server (SLES) 11
-* Ubuntu 14.04
-* Windows Server 2012
+Couchbase Server 6.5 Beta brings many sought-after relational capabilities to the NoSQL world. This release targets improving developer agility, reduced TCO, enhanced manageability, and provides enterprise-grade security and performance. 
 
+include::ROOT:partial$beta-disclaimer.adoc[tag=admonition]
 
-[#supported-platforms-600]
-=== New Supported Platforms
-
-There are no new supported platforms in this release.
-
-See xref:install:install-platforms.adoc[Supported Platforms] for the complete list of supported platforms.
-
-[#known-issues-600]
-=== Known Issues
+Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
-==== Administration/Cluster Management
+// [#release-650]
+// == Release 6.5
```

Comment:
I assume these are commented out because it's the first release, but you may want to include this heading since Antora will complain that you don't have an H2 (right now the first heading on the page is an H3).